.NET/Microsoft.Recognizers.Text.DateTime/Chinese/Parsers/ChineseDateParserConfiguration.cs

```diff
@@ -738,21 +738,15 @@ private int ConvertChineseYearToInteger(string yearChsStr)
         {
             var year = 0;
             var num = 0;
-
-            var regionTitleMatch = this.config.RegionTitleRegex.MatchExact(yearChsStr, trim: true);
-            if (regionTitleMatch != null && regionTitleMatch.Value != string.Empty)
-            {
-                // handle "康熙元年" refer to https://zh.wikipedia.org/wiki/%E5%B9%B4%E5%8F%B7
-                int basicYear = this.config.DynastyYearMap[regionTitleMatch.Value];
-                string biasYearStr = yearChsStr.Substring(regionTitleMatch.Value.Length, yearChsStr.Length - regionTitleMatch.Length);
-                int biasYear = 1;
-                if (biasYearStr != "元")
-                {
-                    biasYear = Convert.ToInt32((double)(numberParser.Parse(integerExtractor.Extract(biasYearStr)[0]).Value ?? 0));
-                }
-
-                year = basicYear + biasYear - 1;
-                return year;
+            int dynastyYear = DateTimeFormatUtil.ParserDynastyYear(yearChsStr,

```

Comment:
Rename to ParseChineseDynastyYear.
Nitpick: Fix code indentation to match current code.

Reply:
Perhaps this method could be in some common Chinese config used by both Date and DatePeriod parsers? 
Then is could be named ParseDynastyYear.

Reply:
Thanks for your suggestions. I have renamed it.